.github/workflows/update-protobufs.yaml

```diff
@@ -0,0 +1,33 @@
+name: Update Protobufs
+on:
+  push:
+    branches:
+    - yaakov/auto-protos # temporary trigger during development
+  #schedule:
+  #- cron: '0 17 * * SUN'
```

Comment:
Just dropping a comment for this to be uncommented once we're ready to merge.

Reply:
Can also add `workflow_dispatch:` to be able to manually trigger this from UI.